ncm-metaconfig/src/main/metaconfig/keepalived/pan/schema.pan

```diff
@@ -39,27 +39,38 @@ type keepalived_service_vrrpinstance_config = {
     'interface' : string
 };
 
-@documentation {
+@documentation{
     The vrrp_instance section
 }
 type keepalived_service_vrrpinstance = {
     'name' : string
     'config' : keepalived_service_vrrpinstance_config
     'virtual_ipaddresses' : keepalived_service_vip[]
-    'track_scripts' : string[]
+    'track_scripts' ? string[]
```

Comment:
Are these paths?
```suggestion
    'track_scripts' ? absolute_file_path[]
```

Reply:
nope, this is the name of a vrrp_script section